zebra-consensus/src/primitives/halo2/tests.rs

```diff
@@ -41,7 +40,7 @@ fn generate_test_vectors() {
     let anchor_bytes = [0; 32];
     let note_value = 10;
 
-    let shielded_data: Vec<zebra_chain::orchard::ShieldedData> = (1..=4)
+    let shielded_data: Vec<zebra_chain::orchard::ShieldedData<OrchardVanilla>> = (1..=4)
```

Comment:
`let shielded_data: Vec<ShieldedData<OrchardVanilla>> = (1..=4)`

Reply:
Subjective: I think `orchard::ShieldedData` would be slightly more readable, though every option seems fine.